### ISSUE TITLE
please make a release! 

### ISSUE THREAD

**Issue by stefson** [gen2brain/volti#68]
hey there, can you please push out a new release? with 0.2.3, I get an error related to dbus-python1.2.4 not recognized, and it seems to have been fixed on the master branche so far. 

thank you!

**Comment by gen2brain** [comment]
Hi, the problem is at that time, after 0.2.3 I added some kind of support for GTK 3, and never finished or tested if it works correctly. In the meantime, I switched to full Qt based DE and not even use `volti` anymore (I use excellent `qastools` now). 

I can release how it is and what it is, but would prefer if someone can confirm it works? with GTK 3, and if there are major issues at least.

**Comment by stefson** [comment]
ok, so I tared up my own v0.2.4 locally. Problem is, it can't find volti.defs:

```
No module named volti.defs
Can't import application modules
Exiting
```

**Comment by gen2brain** [comment]
Did you install that version globally or you are trying to start from source dir? Can you start `python` console and try there just to `from volti.defs import *`. Check if you are trying with same python version. 
There is also this trick here https://github.com/gen2brain/volti/blob/master/volti#L25 that appends current path when trying to start from source dir.

**Comment by stefson** [comment]
volti.gtk* isn't even compiled. Actually I'm on gentoo and used your 9999 ebuild :3

**Comment by gen2brain** [comment]
That is very old ebuild, probably better to use what is current in portage and just add conditional for 9999, there are examples in portage for that.

**Comment by gen2brain** [comment]
I was thinking some time ago that it would be nice to either remove or fix/finish GTK 3 support, and add all patches that are floating around, to have some `stable` version again, but really don't have time to do that.

**Comment by stefson** [comment]
Honestly, you might want to remove gtk3 for now and make it work again first.

**Comment by gen2brain** [comment]
Yes, you are probably right, GTK 3 was a mistake, and I was so disappointed with it at that time that I switched everything to Qt. GTK 2 was and is still a very nice library.

And just remembered, I also added support for official pyalsa bindings, that worked much better then pyalsaaudio, but internal mixer still uses pyalsaaudio so now it needs two different alsa bindings.
